pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -70,7 +70,7 @@ def test_build_table_schema(self, df_schema, using_infer_string):
             "primaryKey": ["idx"],
         }
         if using_infer_string:
-            expected["fields"][2] = {"name": "B", "type": "any", "extDtype": "str"}
+            expected["fields"][2] = {"name": "B", "type": "string", "extDtype": "str"}
```

Comment:
I am wondering if we should still include the `"extDtype": "str"` here for the default string dtype. While having this `extDtype` for actual (opt-in) extension dtypes is very useful to support proper roundtripping, I am not sure it is useful for a default data type.

Reply:
Ah, that is a good point. 
I am wondering for newly written data, if we shouldn't use `"type": "any"` for true object dtype, so we can make that distinction that way.